omnibus/config/software/datadog-agent.rb

```diff
@@ -101,13 +101,17 @@
     # TODO(processes): change this to be ebpf:latest when we move to go1.12.x on the agent
     command "invoke -e process-agent.build --go-version=1.10.1", :env => env
     copy 'bin/process-agent/process-agent', "#{install_dir}/embedded/bin"
-    # We don't use the system-probe in macOS builds
-    if !osx?
-      copy 'bin/system-probe/system-probe', "#{install_dir}/embedded/bin"
-      block { File.chmod(0755, "#{install_dir}/embedded/bin/system-probe") }
-    end
   end
 
+
+  # Build the system-probe
+  if linux?
+    command "invoke -e system-probe.build", :env => env
```

Comment:
Can we add the `--go-version` option to the system-probe task (similar to the one in the process agent task)

and do 
```suggestion
    command "invoke -e system-probe.build --go-version=1.10.1", :env => env
```

(to avoid memory issues we had with `1.11`)

Reply:
Thanks for catching this!

Reply:
Mmm, although this wasn't the case so far for the system-probe build (it was only for the process-agent). Are you sure this is needed?

Reply:
👍  the image the system-probe was built on was using `go 1.10.1`

Reply:
Oh, my bad, makes sense now 👍